docs/shortfin/llm/user/shortfin_with_sglang_frontend_language.md

```diff
@@ -79,57 +76,73 @@ import sglang as sgl
 
 from sglang.lang.chat_template import get_chat_template
 
-backend = sgl.Shortfin(chat_template=get_chat_template("llama-3-instruct"), base_url="http://localhost:8000", ) # Change base_url if running at different address
+backend = sgl.Shortfin(chat_template=get_chat_template("llama-3-instruct"), base_url="http://10.158.231.134:80", ) # Change base_url if running at different address
```

Comment:
Why are we  hardcoding an IP address here? Can we instead pass this in as an argument?

Reply:
Generally better to use X.X.X.X rather than hardcode something that's meaningful to someone for open source docs.

Reply:
Yeah that makes sense. Do you think it would be better to set a local variable with a comment (i.e.):

```python
import sglang as sgl

from sglang.lang.chat_template import get_chat_template

# Set base_url to running shortfin server
base_url = "<BASE_URL>"

backend = sgl.Shortfin(chat_template=get_chat_template("llama-3-instruct"), base_url=base_url) # Change base_url if running at different address

sgl.set_default_backend(backend)

@sgl.function
def multi_turn_question(s, question_1, question_2):
     s += sgl.user(question_1)
     s += sgl.assistant(sgl.gen("answer_1", max_tokens=50))
     s += sgl.user(question_2)
     s += sgl.assistant(sgl.gen("answer_2", max_tokens=50))

state = multi_turn_question.run(question_1="Name the capital city of the USA.", question_2="The Smithsonian is in this location.")

for m in state.messages():
    print(m["role"], m["content"])
```

Or we could add a step to set BASE_URL with an environment variable after the `Shortfin LLM Server` instruction and use that in the code example. That would make it so that they could still copy + paste into interpreter without changes:

```python
import os

import sglang as sgl

from sglang.lang.chat_template import get_chat_template

# Set base_url to running shortfin server
base_url = os.environ["BASE_URL"]

backend = sgl.Shortfin(chat_template=get_chat_template("llama-3-instruct"), base_url=base_url) # Change base_url if running at different address

sgl.set_default_backend(backend)

@sgl.function
def multi_turn_question(s, question_1, question_2):
     s += sgl.user(question_1)
     s += sgl.assistant(sgl.gen("answer_1", max_tokens=50))
     s += sgl.user(question_2)
     s += sgl.assistant(sgl.gen("answer_2", max_tokens=50))

state = multi_turn_question.run(question_1="Name the capital city of the USA.", question_2="The Smithsonian is in this location.")

for m in state.messages():
    print(m["role"], m["content"])
```

Reply:
If the server is running on a different machine than the client yeah, stash it in an environment variable IMO.

Here are some other docs for inspiration:

* https://docs.bentoml.com/en/latest/get-started/cloud-deployment.html#call-the-deployment-endpoint
* https://docs.modular.com/max/tutorials/max-serve-local-to-cloud#3-retrieve-instance-information

Reply:
Added step under `### Shortfin LLm Server` to obtain base_url and store as `SHORTFIN_BASE_URL` environment variable. Updated code examples accordingly